.gitignore

```diff
@@ -3,6 +3,6 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
 .pytest_cache/
 **__pycache__/
```

Comment:
The pattern for excluding `__pycache__` directories should include a leading slash for consistency with other patterns. Update it to `**/__pycache__/` to ensure it correctly matches all `__pycache__` directories.